src/Raven.Server/Exceptions/Attachments/UploadAttachmentException.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+
+namespace Raven.Server.Exceptions.Attachments
+{
+    [Serializable]
+    internal sealed class UploadAttachmentException : Exception
```

Comment:
this needs to be exposed to the client api or not?

Reply:
I think no, since this is thrown inside BackgroundWorkBase